contributing.md

```diff
@@ -1,94 +1,127 @@
-# Contributing to the Windows Community Toolkit
+# Contributing to the Windows Community Toolkit :sparkles::sparkles:
 
-The foundation of the **Windows Community Toolkit** is simplicity. 
+The Windows Community Toolkit team is delighted to welcome you for exhibiting interest in contributing to one of the most exhilarating and growing projects. Any contribution or value added will go a long way to enhance the project.
 
-A developer should be able to quickly and easily learn to use the API. 
+In the next few steps you will be able to see a glimpse of ways you can contribute to the Windows Community Toolkit
 
-Simplicity and a low barrier to entry are must-have features of every API. If you have any second thoughts about the complexity of a design, it is almost always much better to cut the feature from the current release and spend more time to get the design right for the next release. 
+:rotating_light: **It is highly recommended to visit [Windows Community Toolkit Wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki) where you can find complete and detail-oriented content of this page** :rotating_light:
 
-You can always add to an API, you cannot ever remove anything from one. If the design does not feel right, and you ship it anyway, you are likely to regret having done so.
+## <a name="question"></a> Questions :grey_question:
```

Comment:
Let's keep the table of contents as a starting point. e.g.

## Ways to Contribute

There are many ways to contribute to the project besides submitting or fixing issues. Below are a list of links to topics in our Wiki on each subject of how you can get started.

... (TOC from before Questions, Docs, etc...)

A bit more like https://github.com/microsoft/vscode/blob/master/CONTRIBUTING.md

Reply:
I think with the current structure, it looks good now?